packages/auth/src/features/retrievalSteps/sendUserName/handler.ts

```diff
@@ -49,6 +49,12 @@ export default async function sendUserNameHandler(
   const userAgent =
     request.headers['x-real-user-agent'] || request.headers['user-agent']
   if (!PRODUCTION || isDemoUser) {
+    await sendUserName(
```

Comment:
This was not supposed to be here.

Reply:
yeah, actually it came form one of your previous changes , as i had initiated the branch from your branch. Then it remained in this place after merging with feature-email-efa branch. I removed it now. Thanks for the feedback.